File: algoritmos/bubble_sort.py
from typing import List


def bubble_sort(vetor: List[int]) -> List[int]:
    n: int = len(vetor)
    for passnum in range(n-1, 0, -1):
        for i in range(passnum):
            if vetor[i] > vetor[i+1]:
                temp: int = vetor[i]
                vetor[i] = vetor[i+1]
                vetor[i+1] = temp

File: algoritmos/bubbleSort.js
module.exports = {
	bubbleSort: function (array) {
		let houveTroca = true;
		do {
			houveTroca = false;
			for (let j = 0; j < array.length; j++) {
				if (array[j] > array[j + 1]) {
					let temp = array[j];
					array[j] = array[j + 1];
					array[j + 1] = temp;
					houveTroca = true;
				}
			}
		} while (houveTroca);
		return array;
	}
}